### ISSUE TITLE
[Programmatic Access - Make Code - JavaScript] Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.landscape.only.text)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3653]
**User Experience:**
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" control and select it.
6. Verify that user able to access all functionality.
7. Run accessibility insights tool and select fast pass.
8. Verify the issue.

**Actual Result:**
Luminosity ratio is less than 4.5:1 for 'Blocks & JavaScrip' menu items present on 'Header' section.

**Expected Result:**
Luminosity ratio should be more than and equal to 4.5:1 for 'Blocks & JavaScrip' menu items present on 'Header' section.

**Issue:** Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast)

**Target application:** Microsoft MakeCode for micro:bit 

**Element path:** .blocks-menuitem > .landscape.only.text

**Snippet:** <span class="ui text landscape only">Blocks</span>

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 4.08 (foreground color: #9da6c9, background color: #2a3e89, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.3 - Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.4.3; #Win10-Edge(Chromium)

![WCAG1 4 3_JavaScript_CCA](https://user-images.githubusercontent.com/72871089/98805373-789b9800-243d-11eb-8e30-1da57e17c276.jpg)
